mpifileutils.spec

```diff
@@ -250,7 +250,7 @@ done
 %endif
 
 %changelog
-* Mon Feb 07 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
+* Tue Feb 22 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
```

Comment:
Should document here the commit being added now?

Reply:
Updated.